pages/1.13/installing/production/advanced-configuration/configuration-reference/index.md

```diff
@@ -97,6 +97,8 @@ This page contains the configuration parameters for both DC/OS Enterprise and DC
 | [ssh_key_path](#ssh-key-path)                            | The path the installer uses to log into the target nodes. |
 | [ssh_port](#ssh-port)                                    | The port to SSH to, for example 22. |
 | [ssh_user](#ssh-user)                                    | The SSH username, for example `centos`. |
+| [superuser_service_account_uid](#superuser-service-account-uid) | The user ID of the superuser service account |
```

Comment:
Add a "period" at the end of the sentence.

Reply:
Move content to maintain the hierarchy of ordering parameters. Use the following order after `ssh_user`
superuser_password_hash
superuser_service_account_uid
superuser_service_account_public_key